src/Interfaces/ResourceServiceInterface.php

```diff
@@ -382,4 +383,17 @@ public function fetchRefund(Charge $charge, $cancellationId): Cancellation;
      * @throws RuntimeException  A RuntimeException is thrown when there is an error while using the SDK.
      */
     public function fetchShipment($payment, $shipmentId): Shipment;
+
+    /**
+     * Get the configuration for the given payment type.
```

Comment:
What happens if the type does not support the config resource?

Reply:
Then the papi returns an errof object which will cause in an UnzerApiException. I added a note regarding that below.